packages/tus-test/index.html

```diff
@@ -0,0 +1,119 @@
+<html>
```

Comment:
This entire `tus-test` folder should be removed. It was only a local test during development and should not be committed/merged.